integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -115,18 +121,37 @@ class CarbonFileMetastore extends CarbonMetaStore {
     lookupRelation(TableIdentifier(tableName, dbName))(sparkSession)
   }
 
+  val rm = universe.runtimeMirror(getClass.getClassLoader)
+
+  def getField[T: TypeTag : reflect.ClassTag](name: String, obj: T): CatalogTable = {
```

Comment:
Move this method to utility

Reply:
It should be only method exists, all use that method from the utility.

Reply:
Ok